addons/onboarding/models/onboarding_step.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class OnboardingStep(models.Model):
+    _name = 'onboarding.onboarding.step'
```

Comment:
I think "onboarding.step" is sufficient.

We only repeat "onboarding.onboarding" because single-word models are not common and thus are a bit confusing.